bitnami/grafana-loki/templates/gateway/secret.yaml

```
@@ -19,7 +19,5 @@ metadata:
   {{- end }}
 type: Opaque
 data:
-  {{- $password := (include "common.secrets.passwords.manage" (dict "secret" (include "grafana-loki.gateway.fullname" .) "key" "password" "providedValues" (list "gateway.auth.password") "context" $)) }}
-  password: {{ $password | quote }}
-  htpasswd: {{ htpasswd .Values.gateway.auth.username (b64dec $password) | b64enc | quote }}
+  htpasswd: {{ htpasswd .Values.gateway.auth.username .Values.gateway.auth.password | b64enc | quote }}
```

You removed by mistake `password` key in the last commit:
```suggestion
  password: {{ include "common.secrets.passwords.manage" (dict "secret" (include "grafana-loki.gateway.fullname" .) "key" "password" "providedValues" (list "gateway.auth.password") "context" .) }}
  htpasswd: {{ htpasswd .Values.gateway.auth.username .Values.gateway.auth.password | b64enc | quote }}
```

---

No, that was intentional